docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,169 @@
+# ZONE11: Verify SOA MNAME
+
+## Test case identifier
+**ZONE11**
+
+## Table of contents
+
+* [Objective](#Objective)
+* [Scope](#Scope)
+* [Inputs](#Inputs)
+* [Summary](#Summary)
+* [Test procedure](#Test-procedure)
+* [Outcome(s)](#Outcomes)
+* [Special procedural requirements](#Special-procedural-requirements)
+* [Intercase dependencies](#Intercase-dependencies)
+* [Terminology](#terminology)
+
+## Objective
+
+The MNAME field from the SOA record of a zone is supposed to contain the master name server for that zone.
+
+[RFC1035], section 3.3.13, specifies that "the *domain-name* of the name server that was the original or primary source of data for this zone".
+
+[RFC1996], section 2, and [RFC2136], section 1, add that "the primary master is named in the zone's SOA MNAME field and optionally by an NS RR. There is by definition only one primary master server per zone".
+
+Finally, [RFC2181], section 7.2, clarifies that "it is quite clear in the specifications, yet seems to have been widely ignored, that the MNAME field of the SOA record should contain the name of the primary (master) server for the zone identified by the SOA. It should not contain the name of the zone itself. That information would be useless, as to discover it, one needs to start with the domain name of the SOA record - that is the name of the zone".
+ 
+This Test Case will check that:
+ - the MNAME field contains the master name server.
+ - this master name server is authoritative to the zone.
+ - the SOA SERIAL of the MNAME is identical or higher than the ones found from the name servers in the delegation.
+ - the master name server is listed as part of the delegation of the zone.
+
+Note that the SOA SERIAL comparison must follow [RFC1982].
+
+## Scope
+
+It is assumed that *Child Zone* has been tested and reported by [Basic04]. This
+test case will just ignore non-responsive name servers or name servers not
+giving a correct DNS response for an authoritative name server.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Summary
+
+Message Tag                   | Level   | Arguments            | Message ID for message tag
+:---------------------------- |:--------|:---------------------|:---------------------------------------------------------------------------------------
+Z11_MULTIPLE_MNAME            | WARNING | ns_ip_list           | Distinct MNAMEs found, but only one is expected. Returned from name servers "{ns_ip_list}".

```

Comment:
This will overlap message from Consistency06. I think we should avoid having the same message more than once.

Reply:
The user will get the message from Consistency06.